nitro-protocol/test/contracts/ForceMove/conclude.test.ts

```diff
@@ -85,18 +79,15 @@ let channelNonce = getRandomNonce('conclude');
 describe('conclude', () => {
   beforeEach(() => (channelNonce = BigNumber.from(channelNonce).add(1).toHexString()));
   it.each`
-    description | initialFingerprint  | isFinal  | largestTurnNum                   | support        | reasonString
-    ${accepts2} | ${HashZero}         | ${true}  | ${turnNumRecord - 1}             | ${oneState}    | ${undefined}
-    ${accepts2} | ${HashZero}         | ${true}  | ${turnNumRecord + 1}             | ${oneState}    | ${undefined}
-    ${accepts3} | ${channelOpen}      | ${true}  | ${turnNumRecord + 2}             | ${oneState}    | ${undefined}
-    ${accepts5} | ${challengeOngoing} | ${true}  | ${turnNumRecord + 4}             | ${oneState}    | ${undefined}
-    ${accepts6} | ${channelOpen}      | ${true}  | ${turnNumRecord - 1}             | ${oneState}    | ${undefined}
-    ${accepts7} | ${challengeOngoing} | ${true}  | ${turnNumRecord - 1}             | ${oneState}    | ${undefined}
-    ${reverts1} | ${channelOpen}      | ${true}  | ${turnNumRecord + nParticipants} | ${threeStates} | ${MUST_SUBMIT_EXACTLY_1_STATE}
-    ${reverts2} | ${challengeOngoing} | ${true}  | ${turnNumRecord + nParticipants} | ${threeStates} | ${MUST_SUBMIT_EXACTLY_1_STATE}
-    ${reverts3} | ${finalized}        | ${true}  | ${turnNumRecord + 1}             | ${oneState}    | ${CHANNEL_FINALIZED}
-    ${reverts4} | ${HashZero}         | ${false} | ${turnNumRecord - 1}             | ${oneState}    | ${NONFINAL_STATE}
-    ${reverts5} | ${HashZero}         | ${true}  | ${turnNumRecord - nParticipants} | ${threeStates} | ${MUST_SUBMIT_EXACTLY_1_STATE}
+    description | initialFingerprint  | isFinal  | largestTurnNum       | support     | reasonString
```

Comment:
Most of this change is formatting. The actual change is removing cases that hit the revert removed from `_conclude`.